### PR TITLE
Change command "scanapi <spec_file>" to "scanapi run <spec_file>"

### PR DESCRIPTION
**Pull request by abreumatheus**
### Changed
- [BREAKING] - Changed command `scanapi <spec_file>` to `scanapi run <spec_file>`.

### CODE REVIEW

**Comment by abreumatheus** [comment]
Obs: It's failing on `circleci: cangelog-lint` but it doesn't tell me why, so i don't know how to fix it hahaha.

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Renamed `api.(yaml|json)` to `scanapi.yaml` [#222](https://github.com/scanapi/scanapi/issues/20://github.com/scanapi/scanapi/pull/222)
 - Remove top-level `api` key in `scanapi.yaml`. [#231](https://github.com/scanapi/scanapi/pull/231)
 - Renamed `project-name`, `hide-request` and `hide-response` to use underscore. [#228](https://github.com/scanapi/scanapi/issues/228)
+- Changed command `scanapi <spec_file>` to `scanapi run <spec_file>`. [#247](https://github.com/scanapi/scanapi/pull/247)
```
Comment (Pradhvan) [inline CHANGELOG.md:20]: @abreumatheus just a wild guess here can you try 

```
- Renamed change command "scanapi <spec_file>" to "scanapi run <spec_file>" [#247](https://github.com/scanapi/scanapi/pull/247)
```

Comment (Pradhvan) [reply CHANGELOG.md:20]: It's weird that the stack trace of `circleci` dosen't give out any error message.

Comment (barbosa) [reply CHANGELOG.md:20]: Weird 🤔. Diff outputs the same values.

Comment (abreumatheus) [reply CHANGELOG.md]: @Pradhvan @barbosa Changed it, still failing :/

Comment (barbosa) [reply CHANGELOG.md]: @abreumatheus Now you got a different error. See:

```diff
-- Changed command "scanapi <spec_file>" to "scanapi run <spec_file>" [#247](https://github.com/scanapi/scanapi/pull/247) 
+- Changed command "scanapi " to "scanapi run " [#247](https://github.com/scanapi/scanapi/pull/247)
```

Apparently the tool is cutting out everything in between `< >`

Comment (abreumatheus) [reply CHANGELOG.md]: > @abreumatheus Now you got a different error. See:
> 
> ```diff
> -- Changed command "scanapi <spec_file>" to "scanapi run <spec_file>" [#247](https://github.com/scanapi/scanapi/pull/247) 
> +- Changed command "scanapi " to "scanapi run " [#247](https://github.com/scanapi/scanapi/pull/247)
> ```
> 
> Apparently the tool is cutting out everything in between `< >`

Nicely spotted! Will fix it :)

Comment (abreumatheus) [reply CHANGELOG.md]: @barbosa Yeah, it wasn't it. Apparently Circle CI just hates me 🤣

**Comment by abreumatheus** [comment]
@barbosa @Pradhvan Fixed it! Ready to merge!

File: CHANGELOG.md (hunk 1 above)
Comment (Pradhvan) [reply CHANGELOG.md]: @abreumatheus was it just the empty space in the end that was causing this issue ?

Comment (abreumatheus) [reply CHANGELOG.md]: > @abreumatheus was it just the empty space in the end that was causing this issue ? 

Honestly, i dont even know, just ran the linter locally and saved the output to a new file.

Comment (Pradhvan) [reply CHANGELOG.md]: okay. I saw the diff in the end and the only thing I could make out of it was that space xp

Comment (camilamaia) [reply CHANGELOG.md]: wow, this was a nice team effort, what a battle 😂

👏👏👏

**Comment by Pradhvan** [comment]
@camilamaia just a suggestion once the PR is ready to merge we can ask the PR creator to squash all the commits into one commit. That would help us in a cleaner git history too.

**Comment by camilamaia** [comment]
@Pradhvan great suggestion, I squashed and merged it: https://github.com/scanapi/scanapi/commit/725ab68793254cca47f65bea63a096e6c62c5a59

I set this as the default option for merging now for the repo

**Comment by Pradhvan** [comment]
@camilamai yes but if we squash and merge it still gives out 
```
* Changed command `scanapi <spec_file>` to `scanapi run <spec_file>`.

* Added changes to changelog

* Fixed broken test

* Fix changelog lint rules

* Fix changelog lint rules

* Update CHANGELOG.md

* Update CHANGELOG.md

* Update CHANGELOG.md

* Fixed Changelog Lint
```
This commit message is more or less noice for folks who aren't involved in the PR.  

If the PR creator squashes and merges he/she/they can put a single appropriate commit message that can be really helpful for other people too who weren't involved in the PR.( Do correct me if I am wrong) 

PS: I tagged you spectrum thinking the notification won't go as the branch is deleted. So feel free to ignore that. :)

**Comment by camilamaia** [comment]
@Pradhvan hmm that makes sense. We can use this approach in the next PRs! I use a lot —amend command, so I am not that used with squash. Good to know 👍

**Pull request merged**